#This patch file is from Qualcomm Innovation Center, Inc. and is provided under the following license:
#
#Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
#SPDX-License-Identifier: BSD-3-Clause-Clear
commit ceca763f12cb6857f78fa6061c861bcd1cde0703
Author: Rajavenu Kyatham <quic_rkyatham@quicinc.com>
Date:   Thu Nov 21 13:11:47 2024 +0530

    weston: export shared headers

Upstream-Status: Inappropriate [Downstream]pstream-Status: Not Applicable
Signed-off-by: Rajavenu Kyatham <quic_rkyatham@quicinc.com>

diff --git a/shared/meson.build b/shared/meson.build
index 86d46a0..05fbc75 100644
--- a/shared/meson.build
+++ b/shared/meson.build
@@ -1,3 +1,16 @@
+install_headers(
+	'cairo-util.h',
+	'fd-util.h',
+	'file-util.h',
+	'helpers.h',
+	'image-loader.h',
+	'os-compatibility.h',
+	'platform.h',
+	'string-helpers.h',
+	'timespec-util.h',
+	'weston-egl-ext.h',
+	'xalloc.h'
+)
 srcs_libshared = [
	'config-parser.c',
	'option-parser.c',
